SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -191,6 +191,30 @@ void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 		}
 	}
 
+	if (bTrackHandoverProperties)
+	{
+		uint32 Size = 0;
+		uint32 Offset = 0;
+
+		for (FHandoverPropertyInfo& PropertyInfo : Info->HandoverProperties)
+		{
+			if (PropertyInfo.ArrayIdx == 0) // For static arrays, the first element will handle the whole array
+			{
+				// Make sure we conform to Unreal's alignment requirements; this is matched below and in ReplicateActor()
+				Size = Align(Size, PropertyInfo.Property->GetMinAlignment());
+
+				Offset = Align(Offset, PropertyInfo.Property->GetMinAlignment());
+
+				PropertyInfo.ShadowOffset = Offset;
+
+				Offset += PropertyInfo.Property->GetSize();
+				Size += PropertyInfo.Property->GetSize();
```

Comment:
Having both Offset and Size seems redundant here

Reply:
Oh I didn't even read the code when moving it from the ActorChannel - will clean this up.